src/meetings/components/whoIsSpeaking/WhoIsSpeaking.test.tsx

```diff
@@ -70,19 +69,15 @@ beforeEach(() => {
 });
 
 describe('Who is speaking', () => {
-	test('Talking user in central tile is not displayed if the central tile is his video', () => {
-		const spyUseParams = jest.spyOn(ReactRouter, 'useParams');
-		spyUseParams.mockReturnValue({ meetingId: meeting.id });
-		setup(<WhoIsSpeaking visibleTiles={[centralTileVideo]} />);
+	it('Talking user in central tile is not displayed if the central tile is his video', () => {
```

Comment:
`it` and `test` are the same, but we use `test` everywhere so it would be better not to change just one file